dashboard/top-sql.md

```diff
@@ -1,46 +1,145 @@
+
+Top SQL is suitable for analyzing performance issues. The following are some typical Top SQL scenarios:
+
+* You discovered that an individual TiKV instance in the cluster has a very high CPU usage through the Grafana charts. You want to know which SQL statements cause the CPU hotspots so that you can optimize them and better leverage all of your distributed resources.
+* You discovered that the cluster has a very high CPU usage overall and queries are slow. You want to quickly figure out which SQL statements are currently consuming the most CPU resources so that you can optimize them.
+* The CPU usage of the cluster has drastically changed and you want to know the major cause.
+* Analyze the most resource-intensive SQL statements in the cluster and optimize them to reduce hardware costs.
+
+> **Note:**
+>
+> Top SQL currently does not collect the CPU overhead of the SQL statements that were running before the Top SQL feature was enabled. Therefore, it is recommended to [enable the Top SQL feature](#enable-top-sql) as soon as possible instead of enabling it until you have already encountered performance problems.
+
+Top SQL cannot be used to pinpoint non-performance issues, such as incorrect data or abnormal crashes.
+
+The Top SQL feature is still in an early stage and is being continuously enhanced. Here are some scenarios that are **not supported** at the moment:
+
+* Analyzing the overhead of SQL statements outside of Top 5 (for example, when multiple business workloads are mixed).
+* Analyzing the overhead of Top N SQL statements by various dimensions such as users and databases.
+* Analyzing database performance issues that are not caused by high CPU load, such as transaction lock conflicts.
+
+## Access the page
+
+You can access the Top SQL page using either of the following methods:
+
+- After logging into TiDB Dashboard, click **Top SQL** on the left navigation bar.
 
-The Top SQL feature is disabled by default. You can enable the feature for the entire cluster using either of the following methods:
+  ![Top SQL](/media/dashboard/top-sql-access.png)
 
-- Method 1: Log in to TiDB Dashboard, click **Top SQL** in the left pane, click the gear button in the upper-right corner of the page, and then enable the Top SQL feature.
-- Method 2: Set the value of the TiDB system variable [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-new-in-v540) to `ON`.
+- Visit <http://127.0.0.1:2379/dashboard/#/topsql> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+## Enable Top SQL
 
 > **Note:**
 >
-> Enabling Top SQL has a slight impact on the performance of your cluster.
+> To use Top SQL, your cluster should be deployed or upgraded with a recent version of TiUP (v1.9.0 or above) or TiDB Operator (v1.3.0 or above). If your cluster was upgraded from an earlier version, see [FAQ](/dashboard/dashboard-faq.md#a-required-component-ngmonitoring-is-not-started-error-is-shown) for instructions.
```

Comment:
```suggestion
> To use Top SQL, your cluster should be deployed or upgraded with a recent version of TiUP (v1.9.0 or above) or TiDB Operator (v1.3.0 or above). If your cluster was upgraded with an earlier version, see [FAQ](/dashboard/dashboard-faq.md#a-required-component-ngmonitoring-is-not-started-error-is-shown) for instructions.
```

@qiancai I would like to express that if the cluster was upgraded by using earlier version (of TiUP or TiDB Operator). Please take a look again to see whether this `with` or `from` is suitable. Thanks!

Reply:
Got that. I misunderstood it at first. What about updating "If your cluster was upgraded with an earlier version" to "If your cluster was upgraded using an earlier version of TiUP or TiDB Operator"?